README.md

```diff
@@ -48,6 +48,7 @@ Feel free to ask questions in our Matrix channel [#firefoxtranslations:mozilla.o
 - [Reference papers](docs/README.md#references)
 - [Model training guide](docs/training/README.md) - practical advice on how to use the pipeline
 - [High level overview post on Mozilla Hacks](https://hacks.mozilla.org/2022/06/training-efficient-neural-network-models-for-firefox-translations/)
+- [The Training Pipeline DAG](https://docs.google.com/presentation/d/1HkypImI_hbA3n1ljU57ZPAzW8PuQqdv2wrXqj688KtQ/edit?slide=id.g3421e8f521e_1_419#slide=id.g3421e8f521e_1_419)
```

Comment:
Is it possible to download this as an image and include it in the repo?

Reply:
Hmm... I'm a bit hesitant to include it, as we'll need to update it in our drive, and update it in the docs, which is a bit more work. This way it points to the live document which the team has access to edit.